pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -118,17 +189,23 @@ func TestDefaultConfigMap(t *testing.T) {
 					Labels:      map[string]string{"app": api.OpenShiftConsoleName},
 					Annotations: map[string]string{},
 				},
-				Data: map[string]string{configKey: exampleYaml},
+				Data: map[string]string{configKey: exampleYamlWithManagedConfig},
```

Comment:
Yeah I want to know what the `Data` actually will be here. I can see it will `WithManagedConfig` as you are passing in managed config, but I really want this var named what the final output should be, something like `configWithBrandOnline`.

I'm tempted to even just inline the final config...

Reply:
I tweak the names as suggested, let me know if you think we should do more